sources/react-app/src/components/RichText.js

```diff
@@ -20,12 +20,12 @@ export default function (props) {
   const {
     ice,
     model: {
-      content_html_raw
+      content_html
```

Comment:
please double check if this is the right prop. has all the right HTML and such?

Reply:
yes it has the full html, in 3.1.x the `_raw` doesn't come in the response